.github/workflows/ci_test_package.yml

```
@@ -134,5 +132,11 @@ jobs:
           workload_identity_provider: ${{ secrets.GCP_WORKLOAD_IDENTITY_PROVIDER }}
           service_account: ${{ secrets.GCP_SERVICE_ACCOUNT }}
 
+      - name: Character replacement in DBT_VERSION variable for use in schema
+        env:
+          DBT_VERSION: ${{ matrix.version }}
+        run: |
+          echo "DBT_VERSION=${{env.DBT_VERSION//./_}}" >> $GITHUB_ENV
```

Not 100% sure this is the right syntax? From Googling it looks like it should be:

```suggestion
          echo "DBT_VERSION=${DBT_VERSION//./_}" >> $GITHUB_ENV
```